pkg/metrics/metrics.jsx

```diff
@@ -387,16 +390,33 @@ class CurrentMetrics extends React.Component {
                     const is_user = k.match(/^user.*user@\d+\.service.+/);
                     const label = k.replace(/.*\//, '').replace(/\.service$/, '');
                     // only keep cgroup basenames, and drop redundant .service suffix
-                    merged.push([label, v, is_user]);
+                    merged.push([label, v, is_user, false]);
+                }
+                // filter out podman containers
+                const matches = k.match(podmanSystemRe);
+                if (matches && v) {
+                    // truncate to 12 chars like the podman output
+                    const containerid = matches.groups.containerid.substr(0, 12);
+                    const is_user = k.match(/^user.*user@\d+\.service.+/);
```

Comment:
Is that a copy&paste error, or deliberate? user sessions should be handled independently from container scopes.

Reply:
I want to be able to identify user containers so we could possibly link to the them later when cockpit-podman supports some sort of filtering via url query parameters:

User containers appear as the following:
```
'user.slice/user-1000.slice/user@1000.service/user.slice/libpod-30698d4aecca2fa80938b86d5df7b726fd6288e23cb49293eb0b63a9a4c3fdba.scope'
```

Reply:
Makes sense, thanks!